eZ/Publish/API/Repository/Tests/TrashServiceTest.php

```diff
@@ -684,6 +685,51 @@ public function testFindTrashItems()
         $this->assertEquals(4, $searchResult->totalCount);
     }
 
+    /**
+     * @covers  \eZ\Publish\API\Repository\TrashService::findTrashItems
+     *
+     * @depends eZ\Publish\API\Repository\Tests\TrashServiceTest::testTrash
```

Comment:
I would drop `@depends` as at least for me it is kinda troublesome when I try tu run one test only. And I do not see real dependency here.
@alongosz

Reply:
> I would drop `@depends` as at least for me it is kinda troublesome when I try tu run one test only. And I do not see real dependency here.

Yes. Until we have a straightforward way to run single test case w/o passing a chain of unrelated tests in PHPUnit, we should not use it unless the test actually depends on the result returned by dependency.